server/routes/spotifyPlaylist.js

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var router = express.Router();
+
+var getTracksAndCreatePlaylist = require('../modules/spotifyPlaylist.js').getTracksAndCreatePlaylist;
+
+router.get('/new', function(req, res){
+  // songkick API uses res.body.festivalName to get arrayOfArtists
```

Comment:
unfortunately i have bad memory and will forget we havnet doent the songkick api without a comment 😆